src/content/docs/identitymodel-oidcclient/advanced/dpop.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Demonstrating Proof-of-Possession (DPoP)
+description: Learn how to leverage Demonstrating Proof-of-Possession when using OidcClient to build a native OIDC client.
+sidebar:
+  label: DPoP
+  order: 1
+---
+
+[DPoP][dpop-spec] specifies how to bind an asymmetric key stored
+within a JSON Web Key (JWK) to an access token. This will make the access token bound to the key such that if the
+access token were to leak, it cannot be used without also having access to the private key of the corresponding JWK.
+
+The `Duende.IdentityModel.OidcClient.Extensions` library adds supports for DPoP to OidcClient.
+
+## DPoP Key
+
+Before we begin, your application needs to have a DPoP key, in the form of a
+JSON Web Key (or JWK). According to the [DPoP specification][dpop-spec], this
+key needs to use an asymmetric algorithm ("RS", "ES", or "PS" style).
+
+:::note
+This means that the client application is responsible for creating the DPoP key,
+rotating it, and managing its lifetime. For as long as there are access tokens
+(and possibly refresh tokens) bound to a DPoP key, that key needs to remain
+available to the client application.
+:::
+
+Creating a JWK in .NET is pretty straightforward using the `Duende.IdentityModel.OidcClient.Extensions` library:
+
+```csharp
+// Program.cs
+using Duende.IdentityModel.OidcClient.DPoP;
+
+// Creates a JWK using the PS256 algorithm:
+var jwk = JsonWebKeys.CreateRsaJson();
+
+Console.WriteLine(jwk);
+```
+
+:::caution
+In a production scenario, you'll want to store this JWK in a secure location
+and use DataProtection to further protect the JWK.
+:::
+
+## Initializing the OIDC client with DPoP support
+
+Now that we have a JWK, we need to extend the `OidcClientOptions` to configure DPoP:
```

Comment:
Consider adding one or two more sentences explaining the flow here ("options are created, then a JWK is created and using the ConfigureDpop method the OidcClientOptions are configured to use the JWK")